gammapy/image/tests/test_maps.py

```diff
@@ -106,6 +107,47 @@ def test_reproject(self):
         skymap_1_repr = skymap_1.reproject(skymap_2)
         assert_allclose(skymap_1_repr.data, np.full((100, 100), 1))
 
+    def test_lookup_max(self):
+        pos, value = self.skymap.lookup_max() 
+        assert value == 15
+        assert_allclose((359.93, -0.01), (pos.galactic.l.deg, pos.galactic.b.deg))
+
+    def test_lookup_max_region(self):
+        from ...extern.regions.shapes import CircleSkyRegion
+        center = SkyCoord(0, 0, unit='deg', frame='galactic')
+        circle = CircleSkyRegion(center, radius=Quantity(1, 'deg'))
+        pos, value = self.skymap.lookup_max(circle) 
+        assert value == 15
+        assert_allclose((359.93, -0.01), (pos.galactic.l.deg, pos.galactic.b.deg))
+
+    def test_cutout_paste(self):
+        positions = SkyCoord([0, 0, 0, 2, -2], [0, 2, -2, 0, 0],
+                           unit='deg', frame='galactic')
+        BINSZ = 0.02
+
+        # setup coordinate images
+        lon = SkyMap.empty(nxpix=201, nypix=201, binsz=BINSZ)
```

Comment:
Does it help to have 40401 pixels?
Maybe use 11 x 10 pixels or fewer?

This is faster, but more importantly ... if tests fail, very small arrays means it's easier to print the on the console and quickly see what's going on.